### ISSUE TITLE
Verify GPMSA on CASL linear verification problem

### ISSUE THREAD

**Issue by briadam** [libqueso/queso#579]
Verify GPMSA works correctly on Bayes linear verification problem due to Brian Williams and add regression test if possible.

Example is from     

[1] Adams, B.M., Coleman, Kayla, Hooper, Russell W., Khuwaileh, Bassam A., Lewis, Allison, Smith, Ralph C., Swiler, Laura P., Turinsky, Paul J., Williams, Brian J., "User Guidelines and Best Practices for CASL VUQ Analysis Using Dakota, Sandia Technical Report 2016-11614 (also CASL-U-2016-1233-000), September 2016.

**Comment by briadam** [comment]
In the course of doing this, I noticed that (I think) the GPMSA test and examples are scaling simulation and experiment data by variance instead standard deviation.  If that's the case and not right, I can prepare a commit to adjust that...

**Comment by roystgnr** [comment]
Yeah, that appears to be the case to me too, and that's not right.  Good catch!  I apparently copy-and-pasted the scalar example code without reading *nearly* closely enough.
